tests/wbvalve.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+
+from parameterized import parameterized
+
+from utils.channel_access import ChannelAccess
+from utils.ioc_launcher import get_default_ioc_dir
+from utils.test_modes import TestModes
+from utils.testing import get_running_lewis_and_ioc, skip_if_recsim
+
+
+DEVICE_PREFIX = "WBVALVE_01"
+
+emulator_name = "wbvalve"
+
+IOCS = [
+    {
+        "name": DEVICE_PREFIX,
+        "directory": get_default_ioc_dir("WBVALVE"),
+        "macros": {},
+        "emulator": emulator_name,
+    },
+]
+
+
+TEST_MODES = [TestModes.RECSIM, TestModes.DEVSIM]
+
+
+class WbvalveTests(unittest.TestCase):
+    """
+    Tests for the Wbvalve IOC.
+    """
+    def setUp(self):
+        self._lewis, self._ioc = get_running_lewis_and_ioc(emulator_name, DEVICE_PREFIX)
+        self.ca = ChannelAccess(device_prefix=DEVICE_PREFIX)
+        self._lewis.backdoor_run_function_on_device('reset')
+
+    @parameterized.expand(['Julabo 1', 'Julabo 2'])
```

Comment:
This test crashes with an error like this:

```
======================================================================
ERROR [0.000s]: test_GIVEN_an_ioc_WHEN_set_valve_to_wb1on_THEN_status_is_wb1on_1_Julabo_2 (tests.wbvalve.WbvalveTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Instrument\Apps\Python\lib\site-packages\parameterized\parameterized.py", line 392, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Instrument\Apps\EPICS\support\IocTestFramework\master\tests\wbvalve.py", line 39, in test_GIVEN_an_ioc_WHEN_set_valve_to_wb1on_THEN_status_is_wb1on
    self.ca.assert_setting_setpoint_sets_readback(expected_value, 'POS')
  File "C:\Instrument\Apps\EPICS\support\IocTestFramework\master\utils\channel_access.py", line 411, in assert_setting_setpoint_sets_readback
    self.set_pv_value(set_point_pv, value)
  File "C:\Instrument\Apps\EPICS\support\IocTestFramework\master\utils\channel_access.py", line 73, in set_pv_value
    self.ca.set_pv_value(self._create_pv_with_prefix(pv), value, wait=wait, timeout=self._default_timeout)
  File "C:\Instrument\Apps\Python\lib\site-packages\genie_python\genie_cachannel_wrapper.py", line 135, in set_pv_value
    value = CaChannelWrapper.check_for_enum_value(value, chan, name)
  File "C:\Instrument\Apps\Python\lib\site-packages\genie_python\genie_cachannel_wrapper.py", line 284, in check_for_enum_value
    raise InvalidEnumStringException(name, channel_properties["pv_statestrings"])
InvalidEnumStringException: "Invalid string value entered for TE:NDW1799:WBVALVE_01:POS:SP. Valid strings are ('wb1', 'wb2')"
```

This is because the test is attempting to set `Julabo 1` but the IOC expects `wb1`